lib/constants.js

```diff
@@ -4,6 +4,7 @@
 'use strict';
 
 module.exports = {
+  CREDENTIALS_CONTEXT_URL: 'https://w3id.org/credentials',
```

Comment:
This and the related `CredentialIssuance` proof purpose should not be added to this core lib. It should be provided via something like vc.js.

Reply:
Setting aside CredentialIssuance.js here for now: https://gist.github.com/mattcollier/8a2a0e7cb80965c45e03ad8b0cd32fc8